XLS-35d/README.md

```diff
@@ -0,0 +1,284 @@
+# XLS-35d URIToken — Lightweight first-class NFTs for XRPL Protocol Chains
```

Comment:
The folder name should be updated to reflect `XLS-0000d-{title}` where title is lowercase and spaces are replaced with hyphens.